### DISCUSSION TITLE
AWQ vs GPTQ

### DISCUSSION THREAD

**Discussion by kalle07** [oobabooga/text-generation-webui#5309] [Q&A]
share your experience !!!
(win10, RTX4060-16GB)

LOADING AWQ 13B dont work VRAM overload (GPU-Z showes my limit 16GB)
The 13B GPTQ file only uses 13GB and works well

next:
Test on 7B GPTQ(6GB VRAM) 40 tokens/s
Test on 7B AWQ (7GB VRAM) 22 tokens/s

why i should use AWQ ?
is it better in the quality of responses or ???

tell me please  :)

**Comment by ResearchForumOnline** [comment]
ggfu seems to be the only versions that work correctly with linux.
